Automattic/wp-calypso
# Block Editor Tracking: Track details opens (wpcom_block_editor_details_open)

#### Changes proposed in this Pull Request

* Track details button click in header. Event name: `wpcom_block_editor_details_open`

#### Testing instructions



* Follow steps to enable tracking debugging at PCYsg-nrf-p2
* Open Post Editor, click on the Details (i) button in the header. Make sure `wpcom_block_editor_details_open` is fired
* Now click on the button again (while the Details is open), make sure no events are fired.



Related to https://github.com/Automattic/wp-calypso/issues/53410
